typings/pinejs-client-core.d.ts

```diff
@@ -95,14 +102,14 @@ type ExpandedProperty<
 					: never;
 
 export type ExpandResultObject<T, Props extends keyof T> = {
-	[P in Props]: ExpandedProperty<T, P, object>;
+	[P in Props]-?: ExpandedProperty<T, P, object>;
```

Comment:
Now that the reverse navigation properties are always optional this removes their optionality for when they are expanded